api/src/services/users/organization_from_ebiz_poc.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from sqlalchemy import select
+from sqlalchemy.orm import selectinload
+
+import src.adapters.db as db
+from src.db.models.entity_models import SamGovEntity
+from src.db.models.user_models import OrganizationUser, User
+from src.util.sam_gov_utils import link_sam_gov_entity_if_not_exists
+
+logger = logging.getLogger(__name__)
+
+
+def find_sam_gov_entity_for_ebiz_poc(
+    db_session: db.Session, user_email: str
+) -> SamGovEntity | None:
+    """
+    Find a SAM.gov entity where the user's email matches the ebiz POC email.
+
+    Returns the first SAM.gov entity found or None if user is not an ebiz POC.
+    """
+    return (
+        db_session.execute(
+            select(SamGovEntity)
+            .where(SamGovEntity.ebiz_poc_email == user_email)
+            .options(selectinload(SamGovEntity.organization))
+        )
+        .scalars()
+        .first()
+    )
```

Comment:
This should return a list - one person could be the owner of multiple organizations - we'd want to add them all.